apps/files/src/components/AcceptInviteBar.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div id="accept-invite-bar" class="oc-app-bar">
+    <oc-grid flex gutter="small">
+      <div class="uk-flex-1">
+        <div class="uk-flex">
+          <oc-breadcrumb id="files-breadcrumb" :items="breadcrumbs" v-if="showBreadcrumb" home></oc-breadcrumb>
+        </div>
+        <span class="uk-flex uk-flex-middle" v-if="!showBreadcrumb">
```

Comment:
When using `v-if` on same elements directly next to each other or in combination `v-if` - `v-else` - `v-else-if` use `key` to ensure elements will get rendered correctly.

https://vuejs.org/v2/style-guide/#v-if-v-else-if-v-else-without-key-use-with-caution